draft-regext-rdap-extensions.md

```diff
@@ -77,6 +77,7 @@ extension might take the following form:
 
 Though [@!RFC9082] does describe the use of URI query strings, it does not define
 their use with extensions. [@!RFC7480] does instruct servers to ignore unknown query
+parameters. Despite this, there are several extensions that do specify query
```

Comment:
Not clear what we mean here -- if query parameters need to be prefixed with an extension identifier, or not?